README.md

```diff
@@ -7,3 +7,27 @@ Make sure you have docker daemon started.
 mvn clean spring-boot:build-image -DskipTests
 docker run -p 8080:8080 notes:0.0.1-SNAPSHOT
 ```
+
+Run in kubernetes / minikube
+```
+Prerequisite: Create image before run command
+
+minikube start --driver=docker
+minikube -p minikube docker-env after that enter the following:
+SET DOCKER_TLS_VERIFY=1
+SET DOCKER_HOST=tcp://127.0.0.1:32816
+SET DOCKER_CERT_PATH=your value from the command above
+SET MINIKUBE_ACTIVE_DOCKERD=minikube
```

Comment:
I have docker engine and minkube in Ubuntu WSL.
```
minikube start
```
is enough to start a cluster, you can try it.
It also seems like you are working on Windows command line. I suggest to run all commands in WSL, for all of us to be on the same page. A lot of guides also provides Linux commands only